strapi/strapi
# fix(content-releases): remove Refresh button from the details page

### What does it do?

It removes the Refresh button from the Details Release page

### Why is it needed?

Because we need to implement also the Refresh logic before showing the button

### How to test it?

- select a release from the content releases page and open its details page, and then check if the Refresh button is still present
